postgresql/postgresql_installer.sh

```diff
@@ -254,10 +254,10 @@ setup_standard_init()
         read yn
         if [ -z "`echo $yn | egrep -io '^n'`" ]; then
             CREATED_INIT="yes"
-            echo -n "log file: [/usr/local/copperegg/log/revealmetrics_postgresql.log] "
+            echo -n "log file: [/usr/local/copperegg/log/postgresql_metrics.log] "
```

Comment:
This was causing inconsistency for postgresql and couchdb because we somehow implemented different log filenames for /etc/init/ and /etc/init.d based services. These changes will neutralize the logfile name difference.